config/backend-config/internal/cache/cache.go

```diff
@@ -43,28 +43,28 @@ func Start(ctx context.Context, secret [32]byte, key string, ch pubsub.DataChann
 	)
 	// setup db connection
 	dbConn, err = setupDBConn()
+	if err != nil {
```

Comment:
`Start` should return an error as well

Reply:
should we also be closing the pubsub channel..?